dist/less/ds4/variables.less

```diff
@@ -210,6 +210,8 @@
 @ds4-icon-alert-height: 22px;
 @ds4-icon-arrow-down-width: 24px;
 @ds4-icon-arrow-down-height: 13px;
+@ds4-icon-arrow-down-blue-width: 24px;
```

Comment:
Just curious, why are we duplicating the variables for down blue. We can reuse the other ones.
Only thing is if they might change in the future?

Reply:
the pattern that i notice here is the other variables all have duplicate values, so I've just been following that pattern:
```
@ds4-icon-chevron-light-right-width: 13px;
@ds4-icon-chevron-light-right-height: 23px;
@ds4-icon-chevron-light-right-focus-width: 13px;
@ds4-icon-chevron-light-right-focus-height: 23px;
@ds4-icon-chevron-light-right-disabled-width: 13px;
@ds4-icon-chevron-light-right-disabled-height: 23px;
```

Reply:
Agree with @agliga. We should reuses the existing variables no matter what the color of the chevron.

Reply:
@szhou1 Fair point,. That does look like something we might need to clean up in future.